src/app/public/testing/infinite-scroll/infinite-scroll-fixture.spec.ts

```diff
@@ -19,34 +20,44 @@ import {
   SkyInfiniteScrollTestingModule
 } from './infinite-scroll-testing.module';
 
+import {
+  SkyInfiniteScrollComponent
+} from '../../modules/infinite-scroll/infinite-scroll.component';
+
 const DATA_SKY_ID = 'test-infinite-scroll';
 
 //#region Test component
 @Component({
   selector: 'infinite-scroll-fixture',
   template: `
     <div>
-      <button (click)="enabled = !enabled">{{ enabled ? 'Disallow' : 'Allow' }} more</button>
+      <button (click)="hasMore = !hasMore">{{ hasMore ? 'Disallow' : 'Allow' }} more</button>
     </div>
     <ul>
       <li *ngFor="let i of items">{{ i }}</li>
       <li *ngIf="items.length === 0"><em>(no items)</em></li>
     </ul>
     <sky-infinite-scroll
+      #infiniteScrollComponent
       data-sky-id="${DATA_SKY_ID}"
-      [enabled]="enabled"
+      [enabled]="hasMore"
       (scrollEnd)="loadMore()">
     </sky-infinite-scroll>
   `
 })
 class InfiniteScrollTestComponent {
-  public enabled = false;
+  public hasMore = false;
   public items: string[] = [];
+  @ViewChild('infiniteScrollComponent')
+  public infiniteScrollComponent: SkyInfiniteScrollComponent;
```

Comment:
Why do you need to get the `SkyInfiniteScrollComponent? We wouldn't expect our consumers to do that and plus, it's not part of the public API. Do you absolutely need it?

Reply:
Removed the need for the view child reference per our conversation.